src/modules/user/apiKeys/reducer.ts

```diff
@@ -23,13 +25,15 @@ export interface ApiKeysState {
     modal: ApiKeyStateModal;
     pageIndex: number;
     nextPageExists: boolean;
+    error?: CommonError;
 }
 
 export const initialApiKeysState: ApiKeysState = {
     apiKeys: [],
     dataLoaded: false,
     modal: {
         active: false,
+        success: false,
```

Comment:
Why did you add this 'success' property?

Reply:
now I see it's not necessary. I will remove.